tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -952,8 +952,13 @@ void PipelineDumper::updateHashForVertexInputState(const VkPipelineVertexInputSt
 // @param isRelocatableShader : TRUE if we are building relocatable shader
 void PipelineDumper::updateHashForNonFragmentState(const GraphicsPipelineBuildInfo *pipeline, bool isCacheHash,
                                                    MetroHash64 *hasher, bool isRelocatableShader) {
+  auto nggState = &pipeline->nggState;
+  bool enableNgg = nggState->enableNgg;
+
   auto iaState = &pipeline->iaState;
-  hasher->Update(iaState->topology);
+  if (enableNgg) {
```

Comment:
Niggle: prefer no braces for a single-line then block.

Reply:
done.